src/enrichment.cc

```diff
@@ -72,6 +72,7 @@ void Enrichment::Tock() {
   LOG(cyclus::LEV_INFO4, "EnrFac") << prototype() << " used "
                                    << intra_timestep_feed_ << " feed";
   RecordTimeSeries<cyclus::toolkit::ENRICH_FEED>(this, intra_timestep_feed_);
+  RecordTimeSeries<double>("demand"+feed_commod, this, intra_timestep_feed_);
```

Comment:
What is the purpose of the "demand" string here? Is the string prepended with this to create a meaningful data feature in RecordTimeSeries, or would feed_commod be just as good?

Reply:
so the table names for RecordTimeSeries are prepended with either `supply` or `demand` to let `d3ploy` know that this table is for either supply or demand for that commodity. For example, a reactor would record `timeseriesdemanduox` while a source records `timeseriessupplyuox`.

Reply:
Shall we not call that `Request` to be consistent with the DRE's phase name ?

Reply:
I guess not, as `d3ploy` uses `supply` and `demand`...